FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -166,10 +292,21 @@ namespace edm {
     std::vector<std::string> const& pathNames = tns.getTrigPaths();
     std::vector<std::string> const& endPathNames = tns.getEndPaths();
 
+    ConditionalTaskHelper conditionalTaskHelper(
```

Comment:
I can't quite remember, is the one StreamSchedule or one per stream? If the latter, then maybe `ConditionalTaskHelper` could be shared by them all? Depends on if `workerManager_` is shared.